src/modules/task/task.controller.ts

```diff
@@ -1,27 +1,42 @@
 import express, { Request, Response } from 'express';
 import { validate } from '../../validator';
 import { CreateTaskDto } from './dto';
-import { taskService } from './task.service';
-
-const taskController = express.Router();
-
-taskController.get('/', (req: Request, res: Response) => {
-  const result = taskService.findAll();
-
-  res.json(result);
-});
-
-taskController.get('/:id', (req: Request, res: Response) => {
-  const result = taskService.findById(req.params.id);
-
-  res.json(result);
-});
-
-taskController.post('/', (req: Request, res: Response) => {
-  const dto = validate(CreateTaskDto, req.body);
-  const result = taskService.create(dto);
-
-  res.json(result);
-});
-
-export default taskController;
+import TaskRepository from './task.repository';
+import TaskService from './task.service';
+
+export default class TaskController {
+  private router: express.Router;
+  private taskService: TaskService;
+
+  constructor() {
+    this.router = express.Router();
```

Comment:
Роутер будет от BaseController
```suggestion
  private taskService: TaskService;

  constructor() {
```